core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -66,6 +66,7 @@ object PrimOp {
 
 abstract class Arg {
   def localName: String = name
+  private[chisel3] def simpleName: String = name
```

Comment:
```suggestion
  // Same as localName except it can be called on Args before all names are complete (will return ? in those cases)
  // TODO unify with localName which probably shouldn't be calling instanceName on None
  private[chisel3] def earlyLocalName: String = name
```

Good to make the name more precise and leave a comment explaining what this is.